<?xml version="1.0" ?>
<launch>

  <!-- ############################### EXPLORE ############################# -->
  <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
    <remap from="move_base" to="thorvald_001/move_base" />
    <param name="robot_base_frame" value="/thorvald_001/base_link"/>
    <param name="costmap_topic" value="/thorvald_001/move_base/global_costmap/costmap"/>
    <param name="costmap_updates_topic" value="/thorvald_001/move_base/global_costmap/costmap_updates"/>
    <param name="visualize" value="true"/>
    <param name="planner_frequency" value="0.33"/>
    <param name="progress_timeout" value="30.0"/>
    <param name="potential_scale" value="3.0"/>
    <param name="orientation_scale" value="0.0"/>
    <param name="gain_scale" value="1.0"/>
    <param name="transform_tolerance" value="0.3"/>
    <param name="min_frontier_size" value="0.75"/>
    <param name="track_unknown_space" value="true"/>
  </node>

  <!-- ############################### GMAPPING ############################# -->
  <node name="gmapping" pkg="gmapping" type="slam_gmapping">
        <remap from="scan" to="thorvald_001/scan" />
        <param name="odom_frame" type="string" value="thorvald_001/odom"/>
        <param name="base_frame" type="string" value="thorvald_001/base_link"/>
        <param name="map"        type="string" value="/gmap"/>
  </node>     
</launch>






